components/stacks-network/src/chains_coordinator.rs

```diff
@@ -391,8 +391,18 @@ pub async fn start_chains_coordinator(
                     StacksChainEvent::ChainUpdatedWithMicroblocksReorg(_) => {
                         unreachable!() // TODO(lgalabru): good enough for now - code path unreachable in the context of Devnet
                     }
-                    StacksChainEvent::ChainUpdatedWithReorg(_) => {
-                        unreachable!() // TODO(lgalabru): good enough for now - code path unreachable in the context of Devnet
+                    StacksChainEvent::ChainUpdatedWithReorg(data) => {
+                        // looks like in nakamoto we have reorgs, lets try that
+                        let reorg_data = format!("{:#?}", data);
```

Comment:
this will be removed in the final version of the PR
just keeping it to have some logs when testing